CMakeLists.txt

```diff
@@ -120,14 +120,18 @@ else()
   set(PHASAR_STD_FILESYSTEM stdc++fs)
 endif()
 
-option(PHASAR_CONFIG_INSTALL_PREFIX "Customizes the directory, where configuration files for PhASAR are installed (default is $(HOME)/.config)")
+set(PHASAR_CONFIG_INSTALL_PREFIX "" CACHE STRING "Customizes the directory, where configuration files for PhASAR are installed (default is $(HOME)/.config)")
 
-if(${PHASAR_CONFIG_INSTALL_PREFIX} STREQUAL "")
+if("${PHASAR_CONFIG_INSTALL_PREFIX}" STREQUAL "")
   set(PHASAR_CONFIG_INSTALL_DIR "$ENV{HOME}/.config/phasar")
 else()
```

Comment:
(documenting our discussion about this PR on 9 Feb) this hardcodes the building user's home directory in the binary. The config load path should be determined at runtime. there should be a system-wide location for configuration, such as /opt/phasar/config, to which configurations are installed. Users should have the possibility to provide their config file in ~/.config/phasar if wished. I would go for KISS and abort the search for the config file when it is found in the home directory path. We should abstract away the search for a particular config file with a function like getPhasarConfigFilename(std::string &Filename)